includes/sys_form.php

```diff
@@ -21,19 +21,24 @@ function form_hidden($name, $value)
  * @param string $name
  * @param string $label
  * @param int    $value
+ * @param array  $data_attributes
```

Comment:
Even we are in `includes` this is new code. Can you specify the array contents here please?
https://phpstan.org/writing-php-code/phpdoc-types

Reply:
what do you mean? instead of `array` use `string[]` or `array<string, string>`?

Reply:
More the later one i think ^^